### DISCUSSION TITLE
Render and surface display change on framework made all of the information in FPS counter unreadable

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22888] [Q&A] [answered]
[Targeted PR](https://github.com/ppy/osu-framework/pull/5662)

![image](https://user-images.githubusercontent.com/86934170/225991297-f1e81150-1961-4c91-a5d7-a9ce5c81428c.png)

Version: current master

**Comment by peppy** [comment, marked as answer]
I think rather than fixing this osu! side, I'll remove the renderer from the thread name and convey it some other way. This doesn't seem like a good direction.

**Reply by peppy** [reply]
https://github.com/ppy/osu-framework/pull/5686
